src/omnibase_core/models/contracts/subcontracts/model_resource_usage_metric.py

```
@@ -0,0 +1,76 @@
+        description="Critical threshold for this resource",
+        ge=0.0,
+    )
+
+    is_percentage: bool = Field(
+        default=True,
+        description="Whether the usage_value is a percentage (0-100)",
+    )
+
+    @field_validator("usage_value")
+    @classmethod
+    def validate_percentage_range(cls, v: float, info) -> float:
```

Ruff (0.14.3)

Unused class method argument: `info`

(ARG003)